### DISCUSSION TITLE
Welcome to PowerPlatformConnectors Discussions!

### DISCUSSION THREAD

**Discussion by natalie-pienkowska** [microsoft/PowerPlatformConnectors#1000] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Introduce yourselves.
  * Share the connectors you're working on.
  * Find collaborators to work with on connectors.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Laskewitz** [comment]
YAY! 🥳

👋 My name is Daniel Laskewitz. I'm the Power Platform Lead at Sogeti. That's also one of the reasons why I'm involved in the Independent Publisher Connector Program. So far, I've added the following connectors:

- Toggl
- Spotify
- Discord
- Azure AD Invitation Manager
- ChuckNorris.io
- Dad Jokes

Looking forward to seeing more connectors in this space and am excited about what's to come!  🚀

**Comment by ahmadn82** [comment]
Hi!

My name is Ahmad Najjar, and I'm a senior consultant at Infoworker (Oslo, Norway). I'm partner in crime with @Laskewitz in the Independent Publisher Connector Program 😄 I'm working on the following connectors:

- Yelp
- Brønnøysundregistrene (Norwegian entities e-registry)

**Comment by troystaylor** [comment]
Discussions are awesome!

Hi, I'm Troy Taylor and I, too, am part of the Project Coordinator Group, though I have yet to earn my MVP wings. I'm currently a Senior Power Platform Consultant with Hitachi Solutions based out of State College, PA. My connectors for [HubSpot](https://github.com/microsoft/PowerPlatformConnectors/tree/dev/independent-publisher-connectors/HubSpot%20CRM) and Xero were part of the initial wave of Independent Publisher Connectors released/submitted, and I think I've learned quite a lot from each of those developments such that my submitted Etsy connector is even better (though I'm still working on the responses for it). In my backlog, I've had requests from my teammates at Hitachi for: 
- Secure Code Warrior
- GitHub (@matthewdevaney and his Github-stored Markdown for PowerApps idea)
- LastFM (@joel-lindstrom's request)
and I'm sure I'll help with whatever connectors end on the [top ask list](https://github.com/microsoft/PowerPlatformConnectors/tree/dev/independent-publisher-connectors#top-connector-asks).

**Comment by RebekkaAalbers** [comment]
Hi 🙋‍♀️👋!
My name is Rebekka Aalbers and am part of the Project Coordinator group as well. I am a Power Platform Technology Consultant at Macaw, The Netherlands, and Microsoft BizApps MVP.

I am currently not working on any connectors, but supporting the Project Coordinator Group in other ways.

**Comment by nk-gears** [comment]
Hi All,

This is Nirmal from India, a Fulltime Freelancer focussing on architecting Solutions for the cloud (Primarily Azure,Google,AWS). I am always fascinated about automation tools and workflows. I have developed multiple custom connectors at large scale (more than 200 actions) which are still in certification stage. 

Being love with Power Automate Connectors, I am working on converting some of my favourite api list i worked earlier ( https://github.com/nk-gears/api-notes ). Accordingly, I submitted my first "FreeAgent" connector as part of Independent Publisher Connector Programme. Feel free to reach me for any queries related to Custom Connectors Policies as i have used policies extensively in the past.

Also, I am working on building a toolchain for authoring (swagger 2.0) especially for power automate to manage the definitions/path when there are huge actions.

**Reply by natalie-pienkowska** [reply]
Hi Nirmal! Super cool to have received your submission for Free Agent and excited for the toolchain :)

**Comment by sekharmalla** [comment]
## Hello, Everyone! <img src="https://media.giphy.com/media/hvRJCLFzcasrR4ia7z/giphy.gif" width="25px"> This is Chandra 🙋‍♂️.
<br>
I work at an IT company which provide services to 180+ companies in the UAE. I play key role in understanding the business problem and build & providing solutions within Power Platform, Help users Utilize tools they have in Microsoft, automating daily repetitive workflows.
<br>
<br>
I am PL-900 Certified, also the leader of Power Platform User Group leader which (Link in my profile) help the community help each other and share knowledge 
we conduct monthly calls in our community feel free to join. 
<br><br>
Me and @harshinivarma work together she is Power Bi Developer. we both have submitted our first "Notion" in our new Independent Publisher connector program. 
</br> 
<br> 
I love Automation and mostly i integrate My Iphone (shortcuts App) with Power automate for my personal productive purposes like sending My expenses messages to automate then to Power BI etc,. 
<br>
<br>
Finally hoping to help Power Platform community to learn better, grow better, get better.

**Comment by tomka75** [comment]
### Hi everyone and hi video friends,
my name is Tomislav. I´m from Germany, working there as an Office 365 Solution Specialist at deroso Solutions GmbH. Together with my co-host Stefan Riedel I am driving two communities in Hannover - one for Microsoft Collaboration & Cloud themes (MSCCCH) and the other all about Power Platform - Power Platform User Group Hannover (PPUGHaj). Like Stefan, I am an BizApps MVP and I love doing video stuff.

After Microsoft provided the storage of Microsoft Teams meeting recordings into OneDrive and SharePoint, we have the possibility to use the Power Platform, especially Power Automate, to enhance the content management of such recordings. We can move, share, delete, label, retention, ... videos, but currently there is no video production connector. Missing functions for me are:

- Cut the first seconds of the meeting video out
- Concatenate videos, so I can add a preproduced trailer video in front of the meeting video
- Add (still) images to a video for x seconds, for example at the end of a video with more informations where to go after the video
- Add background music
- Add a watermark to the video after the recording and store this as a new video (helpful for sharing)

Someone else interested in such a video production connector? What functions are you missing?

(@natalie-pienkowska - here is my video post - thanks for the hint!)

**Comment by AntonioArantesDevscope** [comment]
Hello, I was asked to move my certified connector files to a new folder, with a new name. I can't figure out how can I do it because my certified connector with the old name has already been merged.
How can I create a new folder and then put the files inside it if I can even create a folder on this repo? Can someone help me here?
